docs/application-setup.md

```diff
@@ -16,23 +16,28 @@ meta -->
 
 # Quick setup
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/kEzTZ2U9KMM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-
-This document shows you how to quickly set up a Luigi web application by installing some of our [examples](https://github.com/SAP/luigi/tree/main/core/examples). 
+<!-- add-attribute:class:warning -->
+> **NOTE:** The quick setup scripts are not intended for production use.  
 
-Luigi can be added to any application using these steps:
-* Adding Luigi's `npm` packages to your project dependencies.
-* Giving Luigi exclusive control over your entry `index.html` file.
-* Starting the server to run your application.
+This document shows you how to quickly set up a Luigi web application by installing some of our [examples](https://github.com/SAP/luigi/tree/main/core/examples). They can be a good way to learn about setting up a Luigi project, but are not intended for production use. How you add Luigi to your application will depend on the framework you use and the application itself. 
 
-In this example, all these steps are achieved through a single **installer** script for any of the frameworks listed below. Keep in mind that if you have a single page application, you need to transfer it to a "two-page application". You should run your application to make sure the existing router doesn't interfere. 
+In the quick setup examples on this page, you can use a single **installer** script which executes these steps: 
+* Add Luigi's `npm` packages (e.g. for Luigi [Core](https://www.npmjs.com/package/@luigi-project/core)/[Client](https://www.npmjs.com/package/@luigi-project/client)) to your project dependencies.
+* Give Luigi exclusive control over your entry `index.html` file.
+* In the case of a single page application, transfer it to a "two-page application". 
```

Comment:
```suggestion
```
This line may be a little too specific that it might be misleading. Consider removing it altogether.